src/main/java/io/appform/dropwizard/actors/actor/ActorConfig.java

```diff
@@ -67,6 +68,13 @@
     @Builder.Default
     private RetryConfig retryConfig = new NoRetryConfig();
 
+    @Builder.Default
+    private boolean queueTTLEnabled = false;
+
+    // TTL in seconds
+    @Builder.Default
+    private Duration ttl = Duration.ofSeconds(1800);
```

Comment:
@santanusinha Shall we just keep a default value for this - Say 30 seconds or a minute and not let this be overridden